src/data/sources/epicgames/EpicGamesGame.vala

```diff
@@ -0,0 +1,172 @@
+/*
+This file is part of GameHub.
+Copyright (C) 2018-2019 Anatoliy Kashkin
+Copyright (C) 2020 Adam Jordanek
+
+GameHub is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+GameHub is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GameHub.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+using Gee;
+using GameHub.Data.DB;
+using GameHub.Utils;
+
+using GameHub.Utils.Downloader;
+
+namespace GameHub.Data.Sources.EpicGames
+{
+	public class EpicGamesGame: Game
+	{
+		public ArrayList<Runnable.Installer>? installers { get; protected set; default = new ArrayList<Runnable.Installer>(); }
+
+		public EpicGamesGame(EpicGames src, string nameP, string idP)
+		{
+			source = src;
+			name = nameP;
+			id = idP;
+			icon = "";
+			image = src.legendary_wrapper.get_image(id);
+			platforms.add(Platform.LINUX);
```

Comment:
Platform is wrong. It should be Windows but by default GameHub use Wine for that.